packages/ses/src/reporting.js

```diff
@@ -59,10 +60,13 @@ export const chooseReporter = reporting => {
     return console;
```

Comment:
If #2934 fixes the problem for reporting = "platform", then returning `console` here should cause the problem #2908 to reappear when using reporting = "console". If we decide to merge this PR into master, that inconsistency becomes an issue.

Reply:
Fixed